docs/scripts/check-closing-quotes.sh.whitelist

```diff
@@ -0,0 +1,10 @@
+# Every line of this file is a regex. Each regex is ignored when checking unpaired quotes.
+# You can add comments to this file if the comment line is unlikely to match.
+# Just do not add too generic patterns (such as empty line) because they would filter out too many lines.
+^      _\\ \\/ _ `/ _ \\/ _  / _ \\/ _ \\\\ \\ /$
+^   _\\ \\/ _ `/ _ \\/ _  / _ \\/ _ \\\\ \\ /$
+^      /    / _ `/ |/ / / _ `/ _ `/ __/ _ \\/ __/$
+^     /    / _ `/ |/ / / _ `/ _ `/ __/ _ \\/ __/$
+^```````````````$
```

Comment:
I think we should change the underlines to a different character.

Reply:
Do you mean in DAML Assistant?

Reply:
Just in the docs where these long sequences of backticks are used to mark headers. Use a different symbol like `~` or `.`. RST pretty much accepts any special character.

Reply:
oh i see! changed to `~`